src/dotnet/commands/dotnet-new/CSharp_MSBuild/$projectName$.csproj

```diff
@@ -3,14 +3,21 @@
 
   <PropertyGroup>
     <OutputType>Exe</OutputType>
-    <TargetFrameworkIdentifier>.NETCoreApp</TargetFrameworkIdentifier>
-    <TargetFrameworkVersion>v1.0</TargetFrameworkVersion>
+    <TargetFrameworks>netcoreapp1.0</TargetFrameworks>
   </PropertyGroup>
 
   <ItemGroup>
     <Compile Include="**\*.cs" Exclude="$(GlobalExclude)" />
```

Comment:
These `Exclude="$(GlobalExclude)"` can be removed now.  They are removed in the dotnet/sdk templates.